src/core/public/chrome/nav_links/nav_links_service.test.ts

```diff
@@ -191,11 +224,12 @@ describe('NavLinksService', () => {
         Object {
           "appUrl": "/app1",
           "baseUrl": "http://localhost/wow/app1",
+          "disabled": false,
           "hidden": true,
           "icon": "legacyApp1",
           "id": "legacyApp1",
           "legacy": true,
-          "order": 5,
+          "order": 0,
```

Comment:
why it changed? could it affect other legacy plugins ?

Reply:
Dataset merge conflict when integrating #50060 as PR merges `availableApps` and `legacyApp` and eli's PR modified the test dataset.
Changed the `legacyApp1` test data to `order: 5,` to return to master state's value instead of modifying the snapshot